### PR TITLE
Fix ocp upgrade in mcg only deployment

### PR DESCRIPTION
**Pull request by vavuthu**
Fixes: #6333

Signed-off-by: vavuthu [vavuthu@redhat.com](mailto:vavuthu@redhat.com)

### CODE REVIEW

**Comment by vavuthu** [comment]
verification job: https://ocs4-jenkins-csb-odf-qe.apps.ocp-c1.prod.psi.redhat.com/job/qe-deploy-ocs-cluster-prod/5107/console

File: ocs_ci/framework/conf/ocp_version/ocp-4.11-ga-config.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+DEPLOYMENT:
+  ocp_url_template: "https://mirror.openshift.com/pub/openshift-v4/clients/ocp/{version}/{file_name}-{os_type}-{version}.tar.gz"
+  installer_version: "4.11-ga"
+  terraform_version: "0.12.26"
+  # ignition_version can be found here
+  # https://docs.openshift.com/container-platform/4.10/post_installation_configuration/machine-configuration-tasks.html#machine-config-overview-post-install-machine-configuration-tasks
+  ignition_version: "3.2.0"
+  ocp_channel: "stable"
+ENV_DATA:
+  vm_template: "rhcos-4.10.3-x86_64-vmware.x86_64"
```
Comment (petr-balogh) [inline ocs_ci/framework/conf/ocp_version/ocp-4.11-ga-config.yaml:16]: We still don't have the templates?  Can you please ask in aos-devel?

File: ocs_ci/framework/conf/ocp_version/ocp-4.11-ga-config.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  acm_hub_channel: release-2.4
```
Comment (prsurve) [inline ocs_ci/framework/conf/ocp_version/ocp-4.11-ga-config.yaml:17]: This needs to be release-2.5

File: ocs_ci/framework/conf/ocp_version/ocp-4.11-ga-minus1-config.yaml [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+DEPLOYMENT:
+  ocp_url_template: "https://mirror.openshift.com/pub/openshift-v4/clients/ocp/{version}/{file_name}-{os_type}-{version}.tar.gz"
+  installer_version: "4.11-ga"
+  terraform_version: "0.12.26"
+  ignition_version: "3.2.0"
+  ocp_channel: "fast"
+  ocp_version_index: -2
+ENV_DATA:
+  vm_template: 'rhcos-4.10.3-x86_64-vmware.x86_64'
+  acm_hub_channel: release-2.4
```
Comment (prsurve) [inline ocs_ci/framework/conf/ocp_version/ocp-4.11-ga-minus1-config.yaml:16]: This needs to be release-2.5

File: ocs_ci/framework/conf/ocp_version/ocp-4.11-ga-config.yaml (hunk 2 above)
Comment (vavuthu) [reply ocs_ci/framework/conf/ocp_version/ocp-4.11-ga-config.yaml:17]: this files shouldn't be here. somehow my branch was messed up. raised https://github.com/red-hat-storage/ocs-ci/pull/6337 and closing this one

File: ocs_ci/framework/conf/ocp_version/ocp-4.11-ga-config.yaml (hunk 1 above)
Comment (vavuthu) [reply ocs_ci/framework/conf/ocp_version/ocp-4.11-ga-config.yaml:16]: this files shouldn't be here. somehow my branch was messed up. raised https://github.com/red-hat-storage/ocs-ci/pull/6337 and closing this one.

not yet. already sent mail for the same.

**Comment by vavuthu** [comment]
somehow my branch was messed up. raised https://github.com/red-hat-storage/ocs-ci/pull/6337 and closing this one

**Pull request closed without merging**